plugin_test.py

```diff
@@ -69,6 +69,31 @@ def test_fail():
     )
 
 
+def test_annotation_third_party_exception(testdir):
+    testdir.makepyfile(
+        my_module="""
+        def fn():
+            raise Exception('oops')
+        """
+    )
+
+    testdir.makepyfile(
+        f"""
```

Comment:
Please avoid using f-string literal. It is not supported in Python 2.7.
```suggestion
        """
```

Reply:
@utgwkk ah, didn't realize that. And this string isn't even templated 🤦. Pushing a fix...